core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -6,91 +6,90 @@
 
 {% block header_js %}
   {{ super() }}
-  {% if collection_id %}
-    <script type="text/javascript">
-      GLOBALS.collectionId = JSON.parse('{{collection_id|js_string}}');
-      GLOBALS.isLoggedIn = Boolean(JSON.parse('{{is_logged_in|js_string}}'));
-    </script>
+  <script ng-if="collection_id" type="text/javascript">
```

Comment:
No, this is not right. the variable ```collection_id``` is received from the backend and you can't use it in ```ng-if``` like that. Also, there is no controller or directive to provide the functionality. 
Although I took a look at the issue, I think what's expected is to simply get rid of ``{% if collection_id %}``` and ```{% endif %}``` lines. Can you please confirm @shubha1593?

Reply:
@durgeshra, as mentioned in #3950 the `{% if colletion_id %}` is not required at all, can you please test this by just removing that jinja from here?